src/FffGcodeWriter.cpp

```diff
@@ -1496,25 +1498,6 @@ bool FffGcodeWriter::processSkinPart(const SliceDataStorage& storage, LayerPlan&
         skin_angle = bridge;
     }
 
-    // add insets
-    if (pattern != EFillMethod::CONCENTRIC)
```

Comment:
That was actually a bug in the original code. I tested it and saw that the skin perimeters were missing when the pattern was concentric.